add image to issue states and triage page (#3386)
getsentry/sentry-docs: src/docs/product/issues/states-triage/index.mdx

## Issue Triage

> Triaging is deciding what do with an issue. The image below shows how issues move through different states, with the applicable labels.

Author: ```suggestion
Triaging is deciding what to do with an issue. The image below shows how issues move through different states, with the applicable labels.
```

> Triaging is the act of deciding what action you want to take on an issue. The image below shows how issues move through different states, with the applicable labels.

Author: ```suggestion
Triaging is deciding what do with an issue. The image below shows how issues move through different states, with the applicable labels.
```
